src/components/PurchaseOrder/POVersionView/OngoingInfoVersionView/OngoingInfoVersionView.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+import { FolioFormattedDate } from '@folio/stripes-acq-components';
+
+import {
+  VersionCheckbox,
+  VersionKeyValue,
+} from '../../../../common/VersionView';
+
+export const OngoingInfoVersionView = ({ version }) => {
+  const data = version?.ongoing;
+
+  return (
+    <>
```

Comment:
Could you please investigate do we need this tag <> at all?

Reply:
react element removed as redundant